### PR TITLE
Add how we QA and how we close stale issues sections

### PR DESCRIPTION
**Pull request by QuincyLarson**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [ ] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [ ] Tested changes locally.
- [ ] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -338,6 +340,25 @@ for further information
...
+
+### How We Review and Merge Pull Requests
+
+Free Code Camp has a team of volunteer Issue Moderators. These Issue Moderators routinely go through open pull requests in a process called [Quality Assurance](https://en.wikipedia.org/wiki/Quality_assurance) (QA).
+
+1. If an Issue Moderator QA's a pull request and confirms that the new code does what it is supposed without seeming to introduce any new bugs, they will comment "LGTM" which means "Looks good to me."
+
+2. Another Issue Moderator will QA the same pull request. Once they have also confirmed that the new code does what it is supposed to without seeming to introduce any new bugs, they will merge the pull request.
+
+If you would like to apply to join our Issue Moderator team - which is a Core Team position - message @berkeleytrue with links to a 5 of your pull requests that have been accepted and 5 issues where you have helped someone else through commneting or QA'ing.
```
Comment (atjonathan) [inline CONTRIBUTING.md]: ~~a 5 of your pull requests~~ 5 of your pull requests

Comment (atjonathan) [inline CONTRIBUTING.md]: ~~else through commneting~~ else through **commenting**

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -338,6 +340,25 @@ for further information
...
+
+
+### How We Close Stale Issues
+
+We will close any issues or pull requests that have been inactive for more than 15 days, except those that match the following criteria:
+- bugs that are confirmed
+- pull requests that are waiting on other pull requests
```
Comment (atjonathan) [inline CONTRIBUTING.md]: ~~pull requests that are waiting on other pull requests~~ pull requests that are waiting on other pull requests **to be merged**

**Review by ghost — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (QuincyLarson) [inline CONTRIBUTING.md]: Fixed.

**Review by QuincyLarson — COMMENTED** [review]
I've made these changes.

**Comment by raisedadead** [comment]
LGTM except for the typo that @atjonathan

File: CONTRIBUTING.md (hunk 2 above)
Comment (QuincyLarson) [inline CONTRIBUTING.md]: Fixed.

**Comment by QuincyLarson** [comment]
@atjonathan I've made all your proposed changes. Please take a look and if it looks good, could you release the review and merge the PR?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -338,6 +340,25 @@ for further information
...
+
+### How We Review and Merge Pull Requests
+
+Free Code Camp has a team of volunteer Issue Moderators. These Issue Moderators routinely go through open pull requests in a process called [Quality Assurance](https://en.wikipedia.org/wiki/Quality_assurance) (QA).
+
+1. If an Issue Moderator QA's a pull request and confirms that the new code does what it is supposed without seeming to introduce any new bugs, they will comment "LGTM" which means "Looks good to me."
+
+2. Another Issue Moderator will QA the same pull request. Once they have also confirmed that the new code does what it is supposed to without seeming to introduce any new bugs, they will merge the pull request.
+
+If you would like to apply to join our Issue Moderator team - which is a Core Team position - message [@berkeleytrue](https://gitter.im/berkeleytrue) with links to 5 of your pull requests that have been accepted and 5 issues where you have helped someone else through commneting or QA'ing.
```
Comment (atjonathan) [inline CONTRIBUTING.md]: As @raisedadead mentioned 😉 
```diff
- else through commneting or QA'ing.
+ else through commenting or QA'ing.
```

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by QuincyLarson** [comment]
@atjonathan @raisedadead thanks for your patience. I fixed that final typo.

**Review by ghost — APPROVED** [review]

**Comment by atjonathan** [comment]
@QuincyLarson, thanks 😃

**Pull request merged**